dev_guide/revhistory_dev_guide.adoc

```diff
@@ -6,6 +6,23 @@
 :experimental:
 
 // do-release: revhist-tables
+== Tue Jun 14 2016
+
+// tag::dev_guide_tue_jun_14_2016[]
+[cols="1,3",options="header"]
+|===
+
+|Affected Topic |Description of Change
+//Tue Jun 14 2016
+
+|link:../dev_guide/compute_resources.html[Quotas and Limit Ranges]
+|Added a section on project resource limits.
+
+|===
+
+// end::dev_guide_tue_jun_14_2016[]
```

Comment:
@ahardin-rh You can take out all of this because the change made to this actually won't show up for online (cuz ifdefs). So pretty much don't need a revhistory for dev_guide this time.